features/FxA-16-devices-view/README.md

```diff
@@ -4,5 +4,22 @@ Device View - Basic UI
 
 https://mozilla.aha.io/features/FXA-16
 
-As A FF user when using Firefox, I want to know the devices and services I am connected to via my Firefox Account, and important information about my the status of my services. For devices, this includes last time synced, record counts of different data types, and optionally data storage size (perhaps an option for self-hosters or others?)
+As a Firefox Accounts user when using Firefox, I want to know the devices and services I am connected to via my Firefox Account, and important information about my the status of my services. For devices, this includes last time synced, and optionally data storage size (perhaps an option for self-hosters or others?)
 
+## Collapsed View
+![Collapsed View](devices1-collapsed.png)
+
+## Minimal View (no rename)
+![Collapsed View](devices6-disconnect-and-refresh-only.png)
```

Comment:
@ryanfeeley I noticed this simple design (devices6-disconnect-and-refresh-only.png) is missing the browser type / version 

Should we include it somehow?

This is the information we have access to:
```
+                device.uaBrowser = session.uaBrowser
+                device.uaBrowserVersion = session.uaBrowserVersion
+                device.uaOS = session.uaOS
+                device.uaOSVersion = session.uaOSVersion
+                device.uaDeviceType = session.uaDeviceType
```